src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/help/Get-AzureRmStreamAnalyticsInput.md

```diff
@@ -109,7 +109,7 @@ This cmdlet does not accept any input.
 
 ## OUTPUTS
 
-### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.StreamAnalytics.Models.PSInput, Microsoft.Azure.Commands.StreamAnalytics]]            Microsoft.Azure.Commands.StreamAnalytics.Models.PSInput
+### Microsoft.Azure.Commands.StreamAnalytics.Models.PSInput            Microsoft.Azure.Commands.StreamAnalytics.Models.PSInput
```

Comment:
@twitchax same comment about multiple outputs in the StreamAnalytic cmdlets (also, the output header for each of these cmdlets has multiple types, so we should trim it to one)

- [`GetAzureStreamAnalyticsInputCommand.cs`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Input/GetAzureStreamAnalyticsInputCommand.cs#L22)
- [`GetAzureStreamAnalyticsJobCommand.cs`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Job/GetAzureStreamAnalyticsJobCommand.cs#L23)
- [`GetAzureStreamAnalyticsOutputCommand.cs`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Output/GetAzureStreamAnalyticsOutputCommand.cs#L22)
- [`GetAzureStreamAnalyticsQuotasCommand.cs`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Subscription/GetAzureStreamAnalyticsQuotasCommand.cs#L23)
- [`GetAzureStreamAnalyticsTransformationCommand.cs`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/Transformation/GetAzureStreamAnalyticsTransformationCommand.cs#L22)